src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/AzureDevOpsTimelineTests.cs

```diff
@@ -45,11 +46,11 @@ public static void Dependencies(IServiceCollection collection)
 
             [ConfigureAllParameters]
             public static Func<IServiceProvider, InMemoryTimelineTelemetryRepository> Repository(
-                IServiceCollection collection,string project, DateTimeOffset latestTime)
+                IServiceCollection collection, string project, string organization, DateTimeOffset latestTime)
```

Comment:
In my head "organization" goes to the left of "project".

Honestly, we really need stronger types here:
* `public record struct DevOpsOrganization(string Name) { public string BaseUrl => ...};`
* `public record struct DevOpsProject(DevOpsOgranization Organization, string Name);`
* `public record struct DevOpsBuild(DevOpsProject Project, int Id);`

That would make these API's not have to get re-written so darn often, and would make it so we don't have to pass 2/3/4 parameters just to talk about a thing. :-)

Reply:
Wait, you already did, it's just named funny. :-)